src/main/frontend/extensions/sci.cljs

```diff
@@ -9,27 +9,33 @@
   (/ (reduce + coll) (count coll)))
 
 (defn eval-string
-  [s]
+  ([s]
+   (eval-string s {}))
+  ([s ns]
   (try
     (sci/eval-string s {:bindings {'sum sum
                                    'average average
                                    'parseFloat js/parseFloat
+                                   'custom-js (if (exists? js/customJs) js/customJs (Empty.))
                                    'isNaN js/isNaN
                                    'log js/console.log
-                                   'pprint util/pp-str}})
+                                   'pprint util/pp-str}
+                        :namespaces ns
```

Comment:
`eval-string` is used in other places besides `<src` so this would be adding this configuration in other features that would be confusing. Let's only pass them as args from `eval-result` for now

Reply:
I found a way to not pass them into `eval-string in last commit

Reply:
My concern wasn't arguments being passed but that other features have additional configuration that are specific to the src feature. I've addressed this